app/views/questionnaires/edit.xls.erb

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0"?>
```

Comment:
We were planning to use this in modifying the error message. Can we push the changes after removing this?